Dockerfile

```diff
@@ -1,13 +1,15 @@
-# Dockerfile to deploy frontend and backend
+# This Dockerfile builds an image that serves both the
+# frontend and the backend
 
-# Build step #1: build the React front end
+# Step #1: build the React front end
 FROM node:14-bullseye-slim as build-step
 WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
 RUN apt-get update && apt-get install -y default-jre wget
 RUN mkdir ./frontend
 COPY ./frontend/package.json ./frontend/package-lock.json ./frontend/
 WORKDIR /app/frontend
+RUN npm install -g npm@latest
```

Comment:
could you pin a specific version (at least major and minor versions) of npm? following the latest version brings unexpected changes from outside, which is often hard to be fixed by ourselves.

Reply:
Fixed. Thanks!

Reply:
Great!